src/components/Login.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="login">
+    <h3>Sign In</h3>
+    <input type="text" v-model="email" placeholder="Email" /><br />
+    <input type="password" v-model="password" placeholder="Password" /><br />
+    <button @click="login">Login</button>
+    <p>
+      You don't have an account ? You can
+      <router-link to="/sign-up">create one</router-link>
+    </p>
+  </div>
+</template>
+
+<script>
+import firebase from "firebase";
+export default {
+  name: "login",
+  data() {
+    return {
+      email: "",
+      password: "",
+    };
+  },
+  methods: {
+    login: function() {
+      firebase
+        .auth()
+        .signInWithEmailAndPassword(this.email, this.password)
+        .then(
```

Comment:
@aboubakar20 Please use `async` `await` instead of `then`.

Reply:
Done